### PR TITLE
[JA] Add Beatmap related pages and resolve some pages' errors

### PR DESCRIPTION
**Pull request by JackTriton0901**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

## Changes

Add `ja.md` for :
- `Beatmap/Beatmap collaborations`
- `Beatmap/Beatmap host`
- `Beatmap/Genre and Language`
- `Beatmap/Guest difficulty`
- `Beatmap/Hype`
- `Beatmap/Marathon`
- `Game mode/osu!taiko`
- `Modding/Kudosu`
- `Modding/Mod for mod`
- `Modding/Mod request`
- `Modding/Modder`
- `Modding/Normal mod`
- `Modding/Timestamp`
- `Ranking Criteria/Difficulty Naming`
- `Ranking Criteria/Scaling BPM`
- `Skinning/Skinner`
- `Skinning/osu!taiko`

Delete duplicated information for `Bit rate` 

Resolve markdown error of `Skinning`
______
Where can I find Japanese reviewer

- @TicClick: add `SKIP_WIKILINK_CHECK`

### CODE REVIEW

**Comment by JackTriton0901** [comment]
Fixed some files

**Comment by JackTriton0901** [comment]
Changed all `\r\n` to `\n` for `Modding/Kudosu`.

**Comment by JackTriton0901** [comment]
Checked on github.dev, `Kudosu` page’s `\r\n` is replaced to `\n`.

**Comment by Walavouchey** [comment]
in the future, please make separate pull requests for related pages. if you intend on getting this reviewed, then grouping everything into one pr will only make it take a while.

don't worry about the line ending errors for now - they can be fixed before merging.

**Comment by TicClick** [comment]
this definitely needs splitting into several small, at least thematic if not per-article pull requests. one more thing, please don't merge the `master` branch into yours, a maintainer will do that when necessary

**Comment by JackTriton0901** [comment]
To fix the link error on `Game mode/osu!taiko`, I have to edit `Gameplay/Grade`.
I'll do this on different pull request.

**Review by peppy — DISMISSED** [review]

**Comment by peppy** [comment]
I've done a prelim check of the translation and it's fine.

File: wiki/Beatmap/Beatmap_host/ja.md
Comment (TicClick) [inline wiki/Beatmap/Beatmap_host/ja.md:7]: "beatmap ranking procedure" needs to be translated -- it's a regular phrase, not a term

File: wiki/Beatmap/Genre_and_language/ja.md
Comment (TicClick) [inline wiki/Beatmap/Genre_and_language/ja.md:10]: image tooltip needs to be translated

File: wiki/Beatmap/Genre_and_language/ja.md
Comment (TicClick) [inline wiki/Beatmap/Genre_and_language/ja.md:18]: image tooltip needs to be translated

**Review by TicClick — CHANGES_REQUESTED** [review]
minor notes, fix this and I'll merge

File: wiki/Beatmap/Beatmap_host/ja.md
Comment (JackTriton0901) [reply wiki/Beatmap/Beatmap_host/ja.md]: translated to ビートマップのランクをさせるために

File: wiki/Beatmap/Genre_and_language/ja.md
Comment (JackTriton0901) [reply wiki/Beatmap/Genre_and_language/ja.md]: translated to ビートマップページに掲載されているジャンルと言語

File: wiki/Beatmap/Genre_and_language/ja.md
Comment (JackTriton0901) [reply wiki/Beatmap/Genre_and_language/ja.md]: translated to ビートマップリストのフィルター

**Review by peppy — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**
